src/shipit_static_analysis/shipit_static_analysis/__init__.py

```diff
@@ -33,8 +49,7 @@ def __hash__(self):
         payload = {
             'class': self.__class__.__name__,
             'path': self.path,
-            'line': self.line,
-            'nb_lines': self.nb_lines,
+            'lines_hash': self.lines_hash,
         }
```

Comment:
Is there a way to add the function name too? If the patch is a copy & paste of some lines with an issue from a function to another, we should show the issue.

Reply:
I don't think it can be done except if supported by the different analyzers themselves.

Reply:
Is it supported in one of the clang ones?

Reply:
No, the only structured output we have is from mozlint, but they don't give this kind of detailed infos